### DISCUSSION TITLE
24 Hour Inactive Crabber cleaning

### DISCUSSION THREAD

**Discussion by thewickersys** [crabber-net/crabber#103] [Ideas]
**Is your feature request related to a problem? Please describe.**
There seems to be clutter of accounts with no molts, name, description, pfp, or banner.

**Describe the solution you'd like**
We were thinking that if an account stays inactive (no molts, name, description, pfp, or banner) for its first 24 hours it gets deleted.

**Describe alternatives you've considered**
An alternative is a bit more strict: even if only one of those things is changed it still gets deleted.

**Additional context**
n/a

**Comment by jakeledoux** [comment]
I see what you mean, but further thought will have to be put into this because if I start deleting accounts automatically I'm going to get hordes of complaints from people who are trying to log into their (now deleted) accounts that they totally intended to use at some point.

It's Murphy's Law applied to UX.
